sos/plugins/logs.py

```diff
@@ -11,26 +11,16 @@
 from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class Logs(Plugin):
+class Logs(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     """System logs"""
 
     plugin_name = "logs"
     profiles = ('system', 'hardware', 'storage')
 
     def setup(self):
-        self.add_copy_spec([
-            "/etc/syslog.conf",
-            "/etc/rsyslog.conf",
-            "/etc/rsyslog.d"
-        ])
-
-        self.add_copy_spec("/var/log/boot.log")
-        self.add_journal(boot="this", catalog=True)
-        self.add_journal(boot="last", catalog=True)
-        self.add_cmd_output("journalctl --disk-usage")
-
         confs = ['/etc/syslog.conf', '/etc/rsyslog.conf']
         logs = []
+        since = "-7days"
```

Comment:
From a RH support perspective, this is a non-starter if we're also dropping the other journal collections. We get cases daily that require going farther back than 7 days, not to mention the cases of "we had an issue like 10 days ago, can you look at this for an RCA now?".